crates/net/network/src/peers/manager.rs

```diff
@@ -725,6 +736,7 @@ impl Peer {
             fork_id: None,
             remove_after_disconnect: false,
             kind: Default::default(),
+            backoff_counter: 1,
```

Comment:
let's start at 0 here

Reply:
just need to increase this before computing the Instant, or we rename this to `next_backoff_counter` to make it obvious that this is the __next__ counter

Reply:
Oh, yeah you're right, will fix. Thanks